account_statement_import_camt/models/account_journal.py

```diff
@@ -1,13 +1,14 @@
 # Copyright 2019 ACSONE SA/NV <thomas.binsfeld@acsone.eu>
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import _, models
+from odoo import models
 
 
 class AccountJournal(models.Model):
     _inherit = "account.journal"
 
     def _get_bank_statements_available_import_formats(self):
+        _ = self.env._
```

Comment:
Don't use **`_`** as a variable in this case. **`_`** is a common convention variable when we need to ignore it in python

Reply:
why not, such variable declaration used to appear in odoo codebase ([example](https://github.com/odoo/odoo/blob/60c6a52b8b9abfbad4fb47b4aaa6f22154877c80/addons/stock/models/stock_rule.py#L386)), and that declaration would save a lot of time replacing from `_` to `self.env._`, I'm just trying to change as little as possible in one MIG commit

Reply:
but you should agree with me that it is still valid. the purpose of declaring variables is reusability and in this case my way will be better and clearer and the code will look neat and less likely to show the small warning `Access to a protected member _... of a class` on some ides.
![image](https://github.com/user-attachments/assets/dfb8c134-4b58-4d8c-9ade-4cb750d43c31)